http-clients/src/test/resources/service-config-example.yml

```diff
@@ -0,0 +1,73 @@
+auth:
+  service: # just a standard ServiceConfig block
+    security:
+      trustStorePath: src/test/resources/trustStore.jks
+    uris:
+      - https://some.internal.url:8643/service/api
+  oauth2:
```

Comment:
we don't necessary need them, but i want include a complete set of configuration to demonstrate it is working as expected and also gives people a idea how service discovery configuration should look like